components/pkg-cfize/habitat/plan.sh

```diff
@@ -39,7 +39,7 @@ do_prepare() {
   p=()
   for tdep in "${pkg_tdeps_resolved[@]}"; do
     if [[ -f "$tdep/PATH" ]]; then
-      p+=($(cat "$tdep/PATH"))
+      mapfile -t p < <(cat "$tdep/PATH")
```

Comment:
I'm not sure about this. I don't really understand what `$tdep/PATH` is. If it's a multiline value that we want to treat each line as a separate element, `mapfile -t` is appropriate, but this invocation will overwrite `p`, unlike the previous code which appended to it.

Reply:
Yep - overlooked that one. Good catch.